samples/uikit/src/main/kotlin/main.kt

```diff
@@ -5,7 +5,7 @@ import platform.UIKit.*
 fun main(args: Array<String>) {
     memScoped {
         val argc = args.size + 1
-        val argv = (arrayOf("konan") + args).map { it.cstr.getPointer(memScope) }.toCValues()
+        val argv = (arrayOf("konan") + args).map { it.cstr.ptr }.toCValues()
```

Comment:
What's the difference between `getPointer(memScope)` and `getPointer(this)`?

Reply:
Those are synonymous: https://github.com/JetBrains/kotlin-native/blob/0a911ed5bbf5413f54facc0b37961cfc3fae0310/Interop/Runtime/src/main/kotlin/kotlinx/cinterop/Utils.kt#L413